components/KitTextInput.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { StyleSheet, TextInput, View, Image } from "react-native";
+import Colors from "../constants/Colors";
+import KitText from "../components/KitText";
+
+export default function KitTextInput(props) {
+  const styles = StyleSheet.create({
+    error: {
+      textAlign: "right",
+      marginTop: 4
+    },
+    image: {
+      marginLeft: 20
+    },
+    statusImage: {
+      marginRight: 20
+    },
+    textInputContainer: {
+      flexDirection: "row",
+      alignItems: "center",
+      backgroundColor: Colors.KIT_LIGHT_GREY,
+      width: 260,
+      height: 40,
+      borderRadius: 20,
+      borderWidth: props.error ? 1 : 0,
+      borderColor: Colors.KIT_RED
+    },
+    textInput: {
+      backgroundColor: Colors.KIT_LIGHT_GREY,
+      fontSize: 16,
+      fontFamily: "poligon-regular",
+      flex: 1,
+      color: Colors.BLACK,
+      paddingLeft: 14
+    }
+  });
+
+  const errorImage = require("../assets/images/redx.png");
+  const successImage = require("../assets/images/greencheck.png");
+  let statusImage;
+  if (props.error) {
+    statusImage = errorImage;
+  } else if (props.success) {
+    statusImage = successImage;
+  }
```

Comment:
props.success is a flag. I originally had props.error as a flag and props.errorMsg as a string separately but then thought it was redundant since having no text as an error message can just indicate no error. do you think it would be better to keep them separate or just rename props.error to props.errorMsg?

Reply:
i chose to just rename props.error to props.errorMsg